src/daemon/daemon.cpp

```diff
@@ -296,6 +355,26 @@ auto try_mem_size(const std::string& val) -> mp::optional<mp::MemorySize>
     }
 }
 
+// TODO can't we keep the bool in the struct with the rest?
```

Comment:
Because we only use it when building the cloud-init. It would be useless later.

Reply:
But it would simplify the code... Even if it's not used later, I think it would be fine to have it as a property of the interface. And it just might be used later if we add new modes. It's all part of the "spec" down to this point anyway so, conceptually, it feels natural to group that info in the `NetworkInterface`.

But ok, I will let you decide. Please remove the TODO if you prefer to keep it as it is.

Reply:
Put the `bool` inside `NetworkInterface`. That simplified the code.